vendor/assets/javascripts/jquery.details.js

```diff
@@ -101,7 +105,9 @@
             // Do the same for the info within the `details` element
             $detailsNotSummary = $details.children(':not(summary)'),
             // This will be used later to look for direct child text nodes
-            $detailsNotSummaryContents = $details.contents(':not(summary)');
```

Comment:
Yes it would. And reads clearer. Have updated and removed the note about the fork it was actually using.